pyright CI jobs are failing

error:

```
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module './impl/format'
Require stack:
- /home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:[19](https://github.com/python/typeshed/actions/runs/8062628621/job/22022768991?pr=11473#step:10:20))
    at require (node:internal/modules/helpers:130:18)
    at /home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:4282:25
    at /home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:4272:17
    at node_modules/jsonc-parser/lib/umd/main.js (/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:4278:7)
    at __require (/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:12:50)
    at Object.<anonymous> (/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:8109:35)
    at Module._compile (node:internal/modules/cjs/loader:1241:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js'
  ]
}

Node.js v[20](https://github.com/python/typeshed/actions/runs/8062628621/job/22022768991?pr=11473#step:10:21).8.1
```

--- a/.github/workflows/meta_tests.yml
+++ b/.github/workflows/meta_tests.yml
@@ -64,7 +64,7 @@ jobs:
           file: "pyproject.toml"
           field: "tool.typeshed.pyright_version"
       - name: Run pyright on typeshed
-        uses: jakebailey/pyright-action@v2.0.2
+        uses: jakebailey/pyright-action@v2
         with:
           version: ${{ steps.pyright_version.outputs.value }}
           python-platform: ${{ matrix.python-platform }}

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -158,14 +158,14 @@ jobs:
           file: "pyproject.toml"
           field: "tool.typeshed.pyright_version"
       - name: Run pyright with basic settings on all the stubs
-        uses: jakebailey/pyright-action@v2.0.2
+        uses: jakebailey/pyright-action@v2
         with:
           version: ${{ steps.pyright_version.outputs.value }}
           python-platform: ${{ matrix.python-platform }}
           python-version: ${{ matrix.python-version }}
           no-comments: ${{ matrix.python-version != '3.11' || matrix.python-platform != 'Linux' }} # Having each job create the same comment is too noisy.
       - name: Run pyright with stricter settings on some of the stubs
-        uses: jakebailey/pyright-action@v2.0.2
+        uses: jakebailey/pyright-action@v2
         with:
           version: ${{ steps.pyright_version.outputs.value }}
           python-platform: ${{ matrix.python-platform }}
@@ -173,7 +173,7 @@ jobs:
           no-comments: ${{ matrix.python-version != '3.11' || matrix.python-platform != 'Linux' }} # Having each job create the same comment is too noisy.
           project: ./pyrightconfig.stricter.json
       - name: Run pyright on the test cases
-        uses: jakebailey/pyright-action@v2.0.2
+        uses: jakebailey/pyright-action@v2
         with:
           version: ${{ steps.pyright_version.outputs.value }}
           python-platform: ${{ matrix.python-platform }}

